### ISSUE TITLE
Breakage on Julia v1.0

### ISSUE THREAD

**Issue by jishnub** [JuliaMath/IntegerMathUtils.jl#5]
https://github.com/JuliaMath/IntegerMathUtils.jl/pull/3/files#diff-28602f93a5d315a5d6dfd2d013de41a6b0bcbd3ed01c7ca21dca919edc4b78fbR13 seems to have broken this package on Julia v1.0, since the macro `@ccall` is not defined. Perhaps the support for v1.0 should be dropped if it isn't supported.

**Comment by oscardssmith** [comment]
Good catch. I will bump support to 1.6 since I don't think many people are still using versions less than the LTS.

**Comment by jishnub** [comment]
I think it'll also be better to update the general registry compat bounds, to make Julia v1.0 use v0.1.0 of this package

**Comment by oscardssmith** [comment]
I was just going to yank 0.1.1, but if there's a better way to do it, that would be great.

**Comment by jishnub** [comment]
Once v0.1.2 is tagged, I can make a PR to the registry

**Comment by devmotion** [comment]
To avoid such problems in the future, IMO it would be good to add CI tests and in particular also test Julia 1.6. It seems currently no tests are run on Github?

(I don't know anything about this package but somehow quite a few notifications showed up in my feed :sweat_smile:)

**Comment by oscardssmith** [comment]
yeah. I do need to get CI set up for this package.
